samples/Samples.Server/Startup.cs

```diff
@@ -98,6 +99,12 @@ public void Configure(IApplicationBuilder app)
                 GraphQLEndPoint = "/graphql",
             });
 
+            app.UseAltairGraphQL(new AltairGraphQLOptions
```

Comment:
It would be nice to use `optons.Headers` in this example to demonstrate some features.

Reply:
What kind of headers do you think I should add? Or would a generic `X-api-token` header be sufficient for the example?

Reply:
Any are suitable just to demonstrate that they can be transmitted.

Reply:
I added an example. Please review.